client.go

```diff
@@ -326,6 +348,10 @@ func (c *Client) get(endpoint endpoint, data interface{}, authFn authFunc) ([]by
 	return c.request(http.MethodGet, endpoint, data, authFn)
 }
 
+func (c *Client) patch(endpoint endpoint, data interface{}, authFn authFunc) ([]byte, error) {
```

Comment:
This doesn't seem to be used in the client and is unrelated to the issue.
Would be better to add this as a separate PR

Reply:
Leftover since the initial implementation was with a `PATCH` method, thanks!